asab/library/providers/zookeeper.py

```diff
@@ -30,9 +36,21 @@ def __init__(self, app, path):
 			config_section_name='',
 			z_path=self.Path
 		)
-		self.Zookeeper = self.ZookeeperContainer.ZooKeeper
+		self.Zookeeper = None
```

Comment:
Not sure if this is a good idea - there is an method that you can call to check if the zookeeper is connected.

Either way, here needs to be a comment that says: this attribute will be set once ZooKeeper is connected.
Or make it "self._zookeeper" (private) so no-one touches that .

Reply:
I cannot check if Zookeeper is connected when the Client does not exist, yet.